packages/website/src/building-blocs/ExampleLayout.tsx

```diff
@@ -56,7 +56,7 @@ export const ExampleLayout: React.FunctionComponent<ExampleLayoutProps> = ({
                 >
                     View source
                 </GithubButton>
-                <h3 className="h1-light normal-white-space">{title}</h3>
+                <h3 className="h1-light normal-white-space @crawled-title">{title}</h3>
```

Comment:
Why use a CSS class starting with a `@`?

Reply:
This is not to be used by the css but by the Coveo Crawler for the website source :smile:

Reply:
What Ariane says☝️

I just thought that if I don't put a special character here, one could see that its not applying any css rules and think that it can be removed